cap/modules/mail/utils.py

```diff
@@ -40,35 +44,70 @@ def path_value_equals(element, JSON):
     return data
 
 
+def get_review_recipients(deposit, config):
+    # mail of owner
+    owner = deposit['_deposit']['owners'][0]
+    owner_mail = User.query.filter_by(id=owner).one().email
+
+    # mail of reviewer
+    reviewer_mail = current_user.email
+
+    recipients = list({owner_mail, reviewer_mail})
+    cadi_id = deposit.get('analysis_context', {}).get('cadi_id')
+
+    if cadi_id:
+        # mail for reviews - Hypernews
+        # should be sent to hn-cms-<cadi-id>@cern.ch if well-formed cadi id
+        if re.match(CADI_REGEX, cadi_id):
+            recipients.append(f'hn-cms-{cadi_id}@cern.ch')
+
+    message = f"Submitted by {owner_mail}, and reviewed by {reviewer_mail}."
+    return message, recipients
+
+
 def get_cms_stat_recipients(record, config):
     data = current_app.config.get("CMS_STATS_COMMITEE_AND_PAGS")
     key = path_value_equals(config.get("path", ""), record)
     recipients = data.get(key, {}).get("contacts")
     params = data.get(key, {}).get("params", {})
 
+    # mail for PDF forum
     pdf_mail = current_app.config.get('PDF_FORUM_MAIL')
     if pdf_mail and record.get('parton_distribution_functions', None):
         recipients.append(pdf_mail)
 
-    # mail for ML surveys
+    # mail for ML surveys - CMS conveners
     conveners_ml_mail = current_app.config.get('CONVENERS_ML_MAIL')
+
+    # Some extra info for the CMS conveners recipients:
+    # 1. if uses centralized CMS ML applications (3.3) (not empty)
+    # 2. if 3.4 = Yes (mva_use = Yes)
+    # 3. if the user adds an app to 3.a (ml_app_use - not empty)
+    # 4. or if the user answers to 3.b (ml_survey.options = Yes)
     centralized_apps = record \
         .get('multivariate_discriminants', {}) \
         .get('use_of_centralized_cms_apps', {}) \
         .get('options', [])
     mva_use = record.get('multivariate_discriminants', {}).get('mva_use')
+    ml_app_use = record.get('ml_app_use', [])
+    ml_survey = record.get('ml_survey', {}).get('options')
 
     if conveners_ml_mail and (
-        (centralized_apps and 'No' not in centralized_apps) or mva_use == 'Yes'
+        (centralized_apps and 'No' not in centralized_apps) or mva_use == 'Yes' or  # noqa
+        ml_app_use or ml_survey == 'Yes'  # noqa
     ):
         recipients.append(conveners_ml_mail)
 
     cadi_id = record.get('analysis_context', {}).get('cadi_id')
 
     message = ""
     if cadi_id:
+        # mail for ML surveys - Hypernews
+        # should be sent to hn-cms-<cadi-id>@cern.ch if well-formed cadi id
+        if re.match(CADI_REGEX, cadi_id):
+            recipients.append(f'hn-cms-{cadi_id}@cern.ch')
```

Comment:
pass as ENV_VAR and fetch from configs...check that email is passed from configuration and if not don't send to hypernews. Variable can be something like `CMS_HYPERNEWS_EMAIL_FORMAT`